microsoft/PowerToys
# [ci][PTRun]Fix signing for new plugins folders

## Summary of the Pull Request

**What is this about:**

After https://github.com/microsoft/PowerToys/pull/14462 , the pipelines were not updated to sign the new path for the PowerToys Run plugins properly.

**What is include in the PR:** 
Update the pipelines with the new plugin binaries paths.

**How does someone test / validate:** 
After this goes into main, wait for the pipelines to build a release candidate and see if they're signed properly.
